src/browser.ts

```diff
@@ -1,10 +1,9 @@
 export { default as ApolloProvider } from './ApolloProvider';
 export { default as graphql, withApollo, InjectedGraphQLProps } from './graphql';
 
-
 // expose easy way to join queries from redux
 export { compose } from 'redux';
 
 // re-exports of close dependencies.
-export { default as ApolloClient, createNetworkInterface, createBatchingNetworkInterface } from 'apollo-client';
+export * from 'apollo-client';
```

Comment:
Will that export the default as ApolloClient? We should make sure that's the case. Otherwise we should just list all the things that we want to export. It's okay if there are a few things missing occasionally, because people can always import directly from apollo-client as a workaround.